projects/Mallard/src/screens/settings/faq-screen.tsx

```diff
@@ -210,9 +210,15 @@ const faqHtml = html`
         You can find out more about subscribing and pricing on our website.
     </p>
     <p>
-        You can also access the Guardian Editions app if you have an existing
-        iTunes subscription. Please note, the Guardian Editions app is no longer
-        available to purchase as a stand alone product in iTunes.
+        "You can also access the Guardian Editions app on compatible iOS devices
```

Comment:
Is the quotation mark at the beginning here a mistake?

Reply:
Yes, it is @aoifemcl15 . Thanks for flagging. I'll remove that.